site/src/pages/tutorials/index.mdx

```diff
@@ -8,17 +8,17 @@ tags:
 
 # Tutorials
 
-Learn how to write a service with Armeria by walking through this tutorial.
-The tutorial takes you through steps based on a sample service, a blog service, to offer you practical guidance.
+Welcome to our Armeria tutorials!
+Learn how to write a service with Armeria by walking through these tutorials.
+These tutorials take you through steps based on a sample service, a blog service, to offer you practical guidance.
 
-## Try running the service
```

Comment:
After taking a look, I agree the previous `Try running the service` section wasn't providing much value.